### PR TITLE
fix: fixed extra space at bottom for list and table panel

### PR DESCRIPTION
**Pull request by SagarRajput-7**
#### Screenshots

| Before  | After |
|---|---|
| image | image |
| image  | image |

### CODE REVIEW

**Review by rahulkeswani101 — APPROVED** [review]

File: frontend/src/container/LogsPanelTable/LogsPanelComponent.styles.scss [hunk 1]
```diff
@@ -63,6 +63,8 @@
 		height: 40px;
 		justify-content: end;
 		padding: 0 8px;
+		margin-top: 12px;
```
Comment (vikrantgupta25) [inline frontend/src/container/LogsPanelTable/LogsPanelComponent.styles.scss:66]: combine them into one `margin` property ?

File: frontend/src/container/GridCardLayout/styles.ts [hunk 2]
```diff
@@ -33,7 +33,14 @@ export const Card = styled(CardComponent)<CardProps>`
 	}
 
 	.ant-card-body {
-		height: calc(100% - 30px);
+		${({ $panelType }): StyledCSS =>
+			$panelType === PANEL_TYPES.TABLE
```
Comment (vikrantgupta25) [inline frontend/src/container/GridCardLayout/styles.ts:37]: the 30px is for legends in other panels ?

Comment (ahmadshaheer) [reply frontend/src/container/GridCardLayout/styles.ts:37]: +1

File: frontend/src/container/LogsPanelTable/LogsPanelComponent.styles.scss (hunk 1 above)
Comment (ahmadshaheer) [reply frontend/src/container/LogsPanelTable/LogsPanelComponent.styles.scss:66]: +1

**Review by ahmadshaheer — APPROVED** [review]

File: frontend/src/container/GridCardLayout/styles.ts (hunk 2 above)
Comment (SagarRajput-7) [reply frontend/src/container/GridCardLayout/styles.ts:37]: Yes, this was the existing space given for legends of other types

File: frontend/src/container/LogsPanelTable/LogsPanelComponent.styles.scss (hunk 1 above)
Comment (SagarRajput-7) [reply frontend/src/container/LogsPanelTable/LogsPanelComponent.styles.scss:66]: Adding changed under this PR - https://github.com/SigNoz/signoz/pull/6056

**Review by YounixM — APPROVED** [review]

**Pull request merged**